### DISCUSSION TITLE
Chip with display

### DISCUSSION THREAD

**Discussion by hodytone** [UkrainianBanderasCat/Digital-Logic-Sim2#20] [Ideas]
I'm trying to create a 7SEGLED with enable pin, but the chip saved with 7SEGLED inside doesn't display anything.

**Comment by UkrainianBanderasCat** [comment]
Are you Ukrainian?

**Reply by hodytone** [reply]
Yep

**Reply by UkrainianBanderasCat** [reply]
Я також

**Reply by hodytone** [reply]
Та ладно?! Серйозно? 😂

**Reply by UkrainianBanderasCat** [reply]
Ага. Я зараз в Іспанії, скоро їду назад у Львів

**Reply by hodytone** [reply]
Взагалі то був сарказм. У тебе дуже цікавий нікнейм і саме через нього я перейшов зі сторінки того ютубера, що розробив цю програму, до твого репозиторію.
Допоможеш мені встановити та налаштувати середовище для розробки?

**Reply by UkrainianBanderasCat** [reply]
Ок, з чим саме помогти? Ми Unity з офіційного сайту, скачуєм бажано новіші версії. Щоб закидувати на GitHub тобі треба скачати Git, в інеті є багато туторіалів

**Reply by hodytone** [reply]
Це потрібен Unity hub?
У нас не буде проблем через використання різних версій?
Що саме я маю робити з ґіт'ом? його десь у налаштуваннях юніті прописувати чи це все окремо працює?

**Reply by hodytone** [reply]
Встановив редактор юніті. Встановив гітхаб десктоп та залогінився.
А далі що? Всі мануали починаються з того, що "створіть свій власний порожній репозиторій та пишіть свій код"
Чи я щось роблю не так?

**Reply by UkrainianBanderasCat** [reply]
Не, ти скачуєш проект з цього репозиторія і відкриваєш його в Юніті, доробляєш щось своє. А потім тобі треба зрообити pull request на мій проект. Pull request це тіп така фігня що ти добавляєш своє, надсилаєш той request і я можу подивитись що ти добавив, якщо все добре, я можу з'єднати твій проект з основним на гітхабі
